experiments/build-google-test.md

```diff
@@ -0,0 +1,14 @@
+# Building the Google Test Applications with Coverage
```

Comment:
Please replace all `build-*.md` files (except `build-swiftshader.md`) with one bash script that builds every project (or one bash script per project).